src/main/java/group1/mips_simulator/Utility.java

```diff
@@ -97,11 +97,26 @@ public static String binaryStrToOctalStr(String binaryStr) {
         return Integer.toOctalString(binaryInt);
     }
 
-    public static String shortToBinaryString_Pretty(short value, int minSize) {
+    public static String shortToBinaryString(short value, int minSize) {
         StringBuilder sb = new StringBuilder(Integer.toBinaryString(0xFFFF & value));
         while (sb.length() < minSize) {
             sb.insert(0, '0');
         }
+        return sb.toString();
+    }
+
+    public static String shortToBinaryString_Pretty(short value, int minSize) {
+        int bitMask = 0b1111_1111_1111_1111;
```

Comment:
not a real note, just didnt know you could use '_' in your bit-ints, thats pretty cool

Reply:
You can use them in regular `int` and when you're defining hex literals `0xFFF_FFF_FFF` for example too